traceloop.go

```diff
@@ -284,7 +284,7 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
-		server.Serve(unixListener)
+		go server.Serve(unixListener)
```

Comment:
I wonder if we should add SIGTERM in addition to SIGINT on line 302. But I see that https://github.com/kinvolk/inspektor-gadget/pull/338 uses SIGINT so this should work.

Reply:
I added `SIGTERM` to be on the safe side.